### ISSUE TITLE
Grafana .debs contain a second copy of the distribution in a tarball installed into /usr/share/grafana/grafana.tar.gz

### ISSUE THREAD

**Issue by raggi** [grafana/grafana#74065]
### What happened?

The debs contains an unpacked copy of a grafana.tar.gz build, as well as a copy of grafana.tar.gz itself in /usr/share/grafana/grafana.tar.gz

### What did you expect to happen?

I'd expect the .deb package to have a single copy of its contents.

### Did this work before?

This appears to be present in 10.0.3 and after. I did not look back further.

### How do we reproduce it?

1. wget https://dl.grafana.com/oss/release/grafana_10.0.3_amd64.deb
2. ar x grafana_10.0.3_amd64.deb
3. tar xf data.tar.gz
4. Observe ./usr/share/grafana/grafana.tar.gz which contains the same binaries and resources as was otherwise unpacked from data.tar.gz.

### Environment (with versions)?

Grafana: 10.0.3, 10.0.4, 10.1.0
OS: Linux

### Grafana platform?

A package manager (APT, YUM, BREW, etc.)

**Comment by kminehart** [comment]
This one is resolved by https://github.com/grafana/grafana-build/pull/143/files#diff-bed996660b8565fb0b20e1f782f62b429038d683197bad47164f60fb884cf574R152. Once we fix the packages in the repository then I'll close this one as well. Thanks again for the reports.

**Comment by kminehart** [comment]
Sincerest apologies, this is now resolved in the latest release (v10.1.1):

```
$ docker run --rm -it debian /bin/sh -c 'apt-get update -yq && apt-get install -y wget && wget https://dl.grafana.com/oss/release/grafana_10.1.1_armhf.deb && dpkg-deb -I grafana_10.1.1_armhf.deb'

 new Debian package, version 2.0.
 size 128390712 bytes: control archive=238899 bytes.
     102 bytes,     3 lines      conffiles
     273 bytes,    12 lines      control
  871271 bytes,  8014 lines      md5sums
    3892 bytes,   102 lines   *  postinst             #!/usr/bin/env
     498 bytes,    17 lines   *  prerm                #!/usr/bin/env
 Package: grafana
 Version: 10.1.1
 License: AGPLv3
 Vendor: "Grafana Labs"
 Architecture: armhf
 Maintainer: contact@grafana.com
 Installed-Size: 378902
 Depends: adduser, libfontconfig1, musl
 Section: default
 Priority: optional
 Homepage: https://grafana.com
 Description: Grafana
 ```
 
 Please let me know if you run into any other issues!
